operations/enterprises.md

```diff
@@ -678,17 +679,24 @@ Adds a new task to the enterprise, optionally to a specified department.
 | `ClientToken` | string | required | Token identifying the client application. |
 | `AccessToken` | string | required | Access token of the client application. |
 | `Client` | string | required | Name and version of the client application. |
-| `DepartmentId` | string | optional | Unique identifier of the [Department](enterprises.md#department) the task is addressed to. |
 | `Name` | string | required | Name \(or title\) of the task. |
 | `Description` | string | optional | Further decription of the task. |
 | `DeadlineUtc` | string | required | Deadline of the task in UTC timezone in ISO 8601 format. |
+| `ServiceOrderId` | string | optional | Unique identifier of the order (for example a [Reservation](reservations.md#reservation) or [Product order](services#add-order)) the task is linked with. |
```

Comment:
Product order is not service order. It has to be product service order. Do we have this distinction in the API docs? CC @vitasamek

Reply:
@siroky I'm aware of that.  But we don't have this distinction in documentation not state that operation `orders/add` creates product service order. I think that in current situation it's unnecessary abstraction; once we have more types of PSO it will make more sense to publish this structure.

Reply:
@jirrick I think, what Honza means, that it is incorrectly described. Product order is not Product Service order. But the result of this endpoint is the Product Service Order Id. Let's discuss it tomorrow in person and discuss how to resolve this in the best way.